From 29c5c8c33a0efed20c82e42323b654e14555621f Mon Sep 17 00:00:00 2001
From: Nic Costa <nic.costa@arm.com>
Date: Wed, 11 Oct 2017 13:31:04 -0500
Subject: [PATCH 1/3] delete unused config macro PAL_USE_FATFS_SD

this option is not used by any underlying library and is confusing
when trying to convert to SPI Flash.

Signed-off-by: Nic Costa <nic.costa@arm.com>
---
 mbed_app.json | 1 -
 1 file changed, 1 deletion(-)

diff --git a/mbed_app.json b/mbed_app.json
index 6a59b44..a313921 100755
--- a/mbed_app.json
+++ b/mbed_app.json
@@ -1,7 +1,6 @@
 {
     "macros": [
         "MBEDTLS_USER_CONFIG_FILE=\"mbedtls_mbed_client_config.h\"",
-        "PAL_USE_FATFS_SD=1",
         "MAX_FIRMWARE_LOCATIONS=2",
         "MAX_COPY_RETRIES=1",
         "SHOW_PROGRESS_BAR=1",
-- 
2.14.2

